### PR TITLE
chore: waf adjustments

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-kswaf-updates-hashicorp.vercel.app/well-architected-framework) 🔎
- [Asana task](https://app.asana.com/0/1202520168081556/1202817201846592) 🎟️

## 🗒️ What

This PR fixes a few random things for the waf rollout. 
- adding the instruqt provider so interactive labs work
- adding a canonical url
- uploading the right landing page asset.

## 🧪 Testing

[visit the landing](https://dev-portal-git-kswaf-updates-hashicorp.vercel.app/well-architected-framework) and other views for a final round of QA before release!

### CODE REVIEW

File: src/lib/get-is-beta-product.ts [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 
 /**
  * Given a `ProductSlug`, returns `true` if the associated Product is a beta
  * product and `false` otherwise. Intended for use in server-side code. If this
  * data is needed client-side, use the more optimized `useIsProductBeta` hook
  * provided by `AllProductDataProvider`.
  */
-const getIsBetaProduct = (productSlug: ProductSlug): boolean => {
+const getIsBetaProduct = (
+	productSlug: ProductSlug | SectionOption
```
Comment (kendallstrautman) [inline src/lib/get-is-beta-product.ts:11]: Since 'waf' isn't a product slug but it will be in the beta product slugs array, had to adjust this.

File: src/views/onboarding/collection-view/index.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export default function OnboardingCollectionView({
 	return (
 		<>
 			<HashiHead>
-				<meta name="robots" content="noindex" />
+				<meta name="robots" content="noindex, nofollow" />
```
Comment (kendallstrautman) [inline src/views/onboarding/collection-view/index.tsx:25]: Fast follow fist for the onboarding work. lmk if I should separate this into a separate PR.

**Review by BRKalow — APPROVED** [review]
LGTM!
